testkit/src/main/resources/logback-test.xml

```diff
@@ -103,5 +103,9 @@
 
     <!-- get rid of "Slf4jLogger started" messages -->
     <logger name="akka.event.slf4j.Slf4jLogger" level="OFF"/>
+    <logger name="akka.event.event.EventStream" level="OFF"/>
```

Comment:
So is this the main `logback-test.xml` file for all of our tests now?

Reply:
Yep!